GGO.Common/Properties/Resources.resx

```diff
@@ -205,4 +205,10 @@
   <data name="GunWTT_SNIP_RIF" type="System.Resources.ResXFileRef, System.Windows.Forms">
     <value>..\Resources\GunWTT_SNIP_RIF.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
   </data>
+  <data name="DeadMarker" type="System.Resources.ResXFileRef, System.Windows.Forms">
+    <value>..\Resources\DeadMarker.png;System.Drawing.Bitmap, System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a</value>
+  </data>
+  <data name="DeadMarker" type="System.Resources.ResXFileRef, System.Windows.Forms">

```

Comment:
Twice?

Reply:
That was weird, I didn't notice that. I fixed this as well though.